### ISSUE TITLE
SSH connection error 'Received disconnect from hostname: 2: Too many authentication failures for username'

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#1722]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=1722
Reporter:kenji.01...@gmail.com Date:2013-05-20 06:47:52 Status:New

<b>What steps will reproduce the problem?</b>
1. ssh connection error &quot;Received disconnect from hostname: 2: Too many authentication failures for username&quot;

<b>What is the expected output?</b>
<b>What do you see instead?</b>

What version of Sequel Pro are you using? 1.0.2
What version of OS X are you using? 10.0.8
What version of MySQL are you connecting to on the server? 5.5.27

<b>Please provide any additional information below.</b>

Since there is too many registered ssh key, it is an error by the number of times of trial. 
Please add ssh option &quot;-o IdentitiesOnly=yes&quot;.

**Comment by vadimgoncharov** [comment]
Guys, any movements on this issue?

**Comment by dmoagx** [comment]
No, but since you can now choose an SSH binary, you could create a simple script file
```bash
#!/bin/bash
/usr/bin/ssh -o IdentitiesOnly=yes "$@"
```
and set that as your ssh client binary in Preferences.

**Comment by vadimgoncharov** [comment]
@dmoagx thanks!
